README.md

```diff
@@ -0,0 +1,30 @@
+# terraform-aws-ecr-repository
+A Terraform module to create an Amazon Web Services (AWS) Elastic Container Registry (ECR) repository.
+
+## Usage
+
+```hcl
+module "ecr-repository" {
+  source  = "azavea/ecr-repository/aws"
+  version = "0.1.0"
+
+  repository_name = "my-containers"
+  
+  attach_lifecycle_policy = true
+  max_number_of_images = 7 # expire untagged images older than 7 days
+}
+```
+
+## Variables
+
+- `repository_name` - name of the repository
+- `attach_lifecycle_policy` - attach a lifecycle policy (default: `false`)
+- `lifecycle_policy` - lifecycle policy JSON (default: `${file("${path.module}/templates/default-lifecycle-policy.json.tpl")}`)
+- `max_number_of_images` - if `attach_lifecycle_policy` is `true` and `lifecycle_policy` has not been set, this variable expires untagged images older than N days (default: `7`)
```

Comment:
This variables leaves me feeling uncomfortable because it:

- Makes it difficult to reason about how it takes effect, depending on the status of `attach_lifecycle_policy` and `lifecycle_policy`
- Gets neglected (but still passed to the template resources) if I override `lifecycle_policy` with a policy that doesn't use count to expire images

I think that we can hardcode 7 (or bump it up to 30) into the default `default-lifecycle-policy.json.tpl` file, and emphasize in its description what the policy does. If we do that, then I think we can remove `max_number_of_images` as a variable.

Reply:
I missed something and accidentally pushed and an old version of `README`. I had renamed `max_number_of_images` to `days_to_retain_images`. Will still remove the functionality.